src/pages/mypage/utils/analytics.ts

```
@@ -0,0 +1,274 @@
+ *
+ * 이벤트 코드: mypage_click_tabImg
+ * - Page: mypage
+ * - Action: click
+ * - Component: tab
+ * - Function: Img
+ *
+ * "생성된 이미지" 탭 클릭 시 전송
+ */
+export const logMyPageClickTabImg = () => {
+  if (!analytics) {
+    console.warn('[Firebase Analytics] Analytics 초기화 실패');
```

ESLint

[error] Replace `'[Firebase·Analytics]·Analytics·초기화·실패'` with `"[Firebase·Analytics]·Analytics·초기화·실패"`

(prettier/prettier)